Marlin/src/pins/stm32f4/pins_ANET_ET4.h

```diff
@@ -199,7 +174,7 @@
 //#define SDIO_SUPPORT
 
 #ifndef SDCARD_CONNECTION
-  #define SDCARD_CONNECTION              ONBOARD
+  #define SDCARD_CONNECTION         CUSTOM_CABLE
```

Comment:
This define doesn't do anything since there's no `CUSTOM_CABLE` setup in the pins file. But there's also no `ONBOARD` either.

Reply:
Yes, it's mysterious. This is what's in the other fork, so if it doesn't matter it can be left as is. If it might matter in future, we can change to a better value.